collect/cache/cache.go

```diff
@@ -142,3 +144,19 @@ func (d *DefaultInMemCache) TakeExpiredTraces(now time.Time) []*types.Trace {
 	}
 	return res
 }
+
+// RemoveSentTraces accepts a set of trace IDs and removes any matching ones from
+// the insertion list. This is used in the case of a cache overrun.
+func (d *DefaultInMemCache) RemoveSentTraces(toDelete map[string]struct{}) {
```

Comment:
The name of this method implies it checks if the trace was sent or not, but it doesn't look like thats the case.  Should we go with `RemoveTraces` since the only criteria is that the trace is present in the `toDelete` map?

Reply:
Good catch; previously, it wasn't a bad name, but this is better.